test/e2ekafka/knative_kafka_test.go

```diff
@@ -151,24 +151,12 @@ func TestKnativeKafka(t *testing.T) {
 		}
 	})
 
-	t.Run("verify Kafka Broker data plane metrics work correctly", func(t *testing.T) {
-		if err := monitoringe2e.VerifyMetrics(caCtx, monitoringe2e.KafkaBrokerDataPlaneQueries); err != nil {
-			t.Fatal("Failed to verify that Kafka Broker data plane metrics work correctly", err)
-		}
-	})
```

Comment:
Why did we drop this test?

Reply:
it's moved to https://github.com/openshift-knative/serverless-operator/blob/main/test/extensione2e/kafka/source_native_kafka_broker_ksvc_test.go#L32-L51 , 

(it requires a broker that actually sends events to have any metrics)

Reply:
Ah missed that, thanks!

Reply:
np!